mm2src/lp_ordermatch/best_orders.rs

```diff
@@ -46,7 +46,10 @@ pub async fn process_best_orders_p2p_request(
         None => return Ok(None),
     };
     let mut result = HashMap::new();
-    let pairs = tickers.iter().map(|ticker| (coin.clone(), ticker.clone()));
+    let pairs = tickers.iter().map(|ticker| match action {
+        BestOrdersAction::Buy => (coin.clone(), ticker.clone()),
```

Comment:
I'm out of the `best_orders` context, and it looks weird why if the user wants to buy, for example, `ETH`, the `ETH` coin will be the `base` coin?

Reply:
Yes, whenever applicable (buy/sell/setprice/best_orders/etc) we consider acting against the base coin. So if users would like to buy ETH they should make the best_orders call with coin `ETH` and action `buy`. Maker orders are always considered as `sell` so we are looking for `ETH/other` pairs. If best_orders is called with `sell` action we should reverse the pair.

Reply:
Got it, thank you!